### ISSUE TITLE
[INSTALL] set build_type=Release for crtm

### ISSUE THREAD

**Issue by AlexanderRichert-NOAA** [JCSDA/spack-stack#827]
Note: Please search to see if an issue already exists for the software you are requesting to be installed.

**Which software in the stack would you like installed?**
`crtm build_type=Release`

**What is the version/tag of the software?**
(current)

**What compilation options would you like set?**
`build_type=Release`

**Installation timeframe:**
1.5.1

**Comment by AlexanderRichert-NOAA** [comment]
Requesting cmake build type 'Release' for crtm per convo with @DavidHuber-NOAA. @climbfuji, it looks like the default build_type in the crtm recipe as of 1.5.0 is RelWithDebInfo. How critical is this? And @DavidHuber-NOAA can you add some detail here on why you want the regular 'Release' build type?

**Comment by climbfuji** [comment]
We should also let @BenjaminTJohnson know about this suggested change.

**Comment by DavidHuber-NOAA** [comment]
@AlexanderRichert-NOAA Yes, CRTM built with `RelWithDebInfo` produces different results from `Release` when run within the GSI for the radiances from the microwave instruments amsua_n15, amsua_n18, amsua_n19, atms_npp, and atms_n20.  Unfortunately, I do not have the regression test comparisons anymore between these two versions, but the differences were quite small as I recall.  That said, I am amenable to keeping `RelWithDebInfo` if there is a good reason for it.

**Comment by BenjaminTJohnson** [comment]
@AlexanderRichert-NOAA sorry, just want to verify that this is CRTM v2.4.0, correct?

**Comment by AlexanderRichert-NOAA** [comment]
Correct-- spack-stack currently uses 2.4.0, then assuming we made this change, we would keep the setting moving forward.

**Comment by climbfuji** [comment]
PR https://github.com/spack/spack/pull/41163 was just merged into spack develop. This PR changes the build type for CRTM to release. We should cherry-pick the PR (or at least the CRTM part of it) after #884 is merged.

**Comment by climbfuji** [comment]
This still needs to be done for releasde/1.6.0 (on release branch).

**Comment by climbfuji** [comment]
Closed via https://github.com/JCSDA/spack-stack/pull/919
